tools/helper/package.json

```diff
@@ -31,6 +31,7 @@
     "./shared": "./lib/shared/index.js",
     "./noopComponent": "./lib/client/noopComponent.js",
     "./noopModule": "./lib/client/noopModule.js",
+    "./normalize": "./lib/client/styles/normalize.css",
```

Comment:
I believe the `.css` suffix should not be omitted here; it should clearly indicate that this is a `css` file.